install/helm/open-match/values.yaml

```diff
@@ -33,35 +33,41 @@ openmatch:
     reportingPeriod: "5s"
   backend:
     install: true
+    replicas: 3
     grpc:
       port: 50505
     http:
       port: 51505
   frontend:
     install: true
+    replicas: 3
     grpc:
       port: 50504
     http:
       port: 51504
   mmlogic:
     install: true
+    replicas: 3
     grpc:
       port: 50503
     http:
       port: 51503
   synchronizer:
     install: true
+    replicas: 1
```

Comment:
What might be helpful is to add a comment here that indicates why this is 1 (and should be 1 for now) to prevent someone in future accidentally assuming this was a typo (since all others are 3) and setting it to 3.